pkg/kubelet/config/flags.go

```diff
@@ -57,6 +57,7 @@ func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 
 	// Docker-specific settings.
 	fs.StringVar(&s.PodSandboxImage, "pod-infra-container-image", s.PodSandboxImage, fmt.Sprintf("Specified image will not be pruned by the image garbage collector. CRI implementations have their own configuration to set this image."))
+	fs.MarkDeprecated("pod-infra-container-image", "will be removed in 1.27 as it is replaced by pinned images")
```

Comment:
"pod-infra-container-image", "will be removed in 1.27. Image garbage collector will get sandbox image information from container runtimes")

Reply:
Thanks. Done.